client/state/activity-log/log/is-discarded/index.js

```diff
@@ -0,0 +1,52 @@
+/** @format */
+/**
+ * External dependencies
+ */
+
+import { memoize } from 'lodash';
+
+/**
+ * Normalize timestamp values
+ *
+ * Some timestamps are in seconds instead
+ * of in milliseconds and this will make
+ * sure they are all reported in ms
```

Comment:
working on it, but this is an easy enough workaround - it's always questionable whether to leave something like this in. it's cheap, and time issues pop up all the time, especially since JS decided to report time in `ms` unlike every-other-language…